test/Microsoft.AspNet.Mvc.Formatters.Json.Test/JsonOutputFormatterTests.cs

```diff
@@ -284,6 +286,13 @@ public void Constructor_UsesSerializerSettings()
             return encoding;
         }
 
+        private static ILogger GetLogger()
+        {
+            var logger = new Mock<ILogger>();
```

Comment:
BTW we do have a TestLogger in the Logger.Testing project. Would that help here?